test-case/check-audio-equalizer.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+##
+## Case Name: check-audio-equalizer.sh
+## Preconditions:
+##    SOF Topology should have EQ component(s) in any pipeline
+## Description:
+##    check IIR/FIR config working
+## Case step:
+##    1. TODO: Check if the topology has a EQ component in it
+##    2. Test with IIR config list
+##    3. Test with FIR config list
+## Expect result:
+##    sof-ctl and aplay should return 0
+##
+
+# source from the relative path of current folder
+source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+
+OPT_OPT_lst['t']='tplg'     OPT_DESC_lst['t']='tplg file, default value is env TPLG: $TPLG'
+OPT_PARM_lst['t']=1         OPT_VALUE_lst['t']="$TPLG"
+
+
+# Test equalizer
+func_test_eq()
+{
+    local id=$1
+    local conf=$2
+    dlogc "sof-ctl -Dhw:0 -n $id -s $conf"
+    sof-ctl -Dhw:0 -n $id -s $conf
+    if [ $? -ne 0 ]; then
+        dloge "Equalizer setting failure with $conf"
+        return 1
+    fi
+    # test with aplay
+    dlogc "aplay -Dhw:0,0 -f dat -c 2 /dev/zero -d 5"
```

Comment:
It is better not to use the static card and pcm number.

Reply:
@fredoh9 Could you use some scripts to get the PCM with eq here.